source/ide/ide_methods.bas

```diff
@@ -19801,9 +19828,19 @@ FUNCTION findHelpTopic$(topic$, lnks, firstOnly AS _BYTE)
     'check if topic$ is in help links
     '    - returns a list of help links separated by CHR$(0)
     '    - returns the total number of links found by changing 'lnks'
+    IF NOT _FILEEXISTS("internal\help\links.bin") THEN
+        q$ = ideyesnobox("Help problem", "The help system is not yet initialized,\ndo it now? (Make sure you're online.)")
+        PCOPY 3, 0: SCREEN , , 3, 0
+        IF q$ = "N" THEN lnks = 0: lnks$ = CHR$(0): GOTO noLinksFile
+        Help_IgnoreCache = 1
+        a$ = Wiki$("Keyword Reference - Alphabetical")
+        Help_IgnoreCache = 0
+        Help_ww = 78: WikiParse a$ 'assume standard IDE width for parsing
+    END IF
+    '----------
     a2$ = UCASE$(topic$)
     fh = FREEFILE
-    OPEN "internal\help\links.bin" FOR BINARY AS #fh
+    OPEN "internal\help\links.bin" FOR INPUT AS #fh

```

Comment:
Is there a reason `BINARY` didn't work? From my understanding it should be faster and functionally work the same in this situation.